client/src/mock/api/expense/expenses.ts

```diff
@@ -89,6 +131,30 @@ export const mockDeleteTransaction = (
 	return response(500) as DeleteTransactionResponse;
 };
 
+function loadExpenses() {
+	loadTransactions();
+	if (!expenses) {
+		expenses = expensesJson.map((expense) => {
+			return {
+				id: expense.id,
+				name: expense.name,
+				description: expense.description,
+				recurrenceType: recurrenceTypes.find(
+					(recurrenceType) => recurrenceType.id === expense.recurrence_type_id
+				),
+				currencyType: currencyTypes.find(
+					(currencyType) => currencyType.id === expense.currency_type_id
+				),
+				predefinedExpenseId: expense.predefined_expense_id,
+				startDate: expense.startDate,
+				value: expense.value,
+				userId: expense.user_id,
+				transactions: [],
+			} as Expense;
```

Comment:
You either could reuse a new function mentioned above or use spread operator (...) to overwrite only recurrenceType, currencyType and transactions.
(predefined_expense_id probably should be renamed in the json, to avoid mapping snake_case to pascalCase)

Reply:
Same as a above.